debian/changelog

```diff
@@ -12,7 +18,7 @@ opx-nas-interface (5.18.0+opx9) unstable; urgency=medium
 
 opx-nas-interface (5.18.0+opx8) unstable; urgency=medium
 
-  * Bugfix: opx-vrf-config script so success message isn't display 
+  * Bugfix: opx-vrf-config script so success message isn't display
```

Comment:
Please get rid of the extra (space) changes in this file.

Reply:
These are actually getting rid of the extra-spaces at the end of the line. It's the correct thing to do.